RadixWallet/Core/SharedModels/Entities/EntitiesControlledByFactorSource.swift

```diff
@@ -24,6 +24,55 @@ public struct EntitiesControlledByFactorSource: Sendable, Hashable, Identifiable
 }
 
 extension EntitiesControlledByFactorSource {
+	public struct PerCurve: Equatable, Sendable {
+		public struct ID: Sendable, Hashable {
```

Comment:
This is a "compound ID", allowing for us to show the same Device Factor Source (Mnemonic) **twice** in the Seed Phrases list, once with Babylon accounts, and once with Olympia Accounts, **without doing large edits to the Reducer, which is `IdentifiedArrayOf` based** (as most reducers showing lists).

Reply:
I think we are overusing `IdentifiedArray`, it only really serves a purpose when you want to lookup or replace an element using its id. ForEach only requires that the elements are identifiable, so it's not needed for that.